android/src/main/java/com/google/android/react/navsdk/NavModule.java

```diff
@@ -688,40 +693,40 @@ private void showNavInfo(NavInfo navInfo) {
-      if (navInfo.getTimeToCurrentStepSeconds() != null)
-        map.putInt("timeToCurrentStepSeconds", navInfo.getTimeToCurrentStepSeconds());
-      if (navInfo.getTimeToFinalDestinationSeconds() != null)
-        map.putInt("timeToFinalDestinationSeconds", navInfo.getTimeToFinalDestinationSeconds());
-      if (navInfo.getTimeToNextDestinationSeconds() != null)
-        map.putInt("timeToNextDestinationSeconds", navInfo.getTimeToNextDestinationSeconds());
-      if (navInfo.getCurrentStep() != null)
-        map.putMap(
-            "currentStep", ObjectTranslationUtil.getMapFromStepInfo(navInfo.getCurrentStep()));
-
-      WritableArray remainingSteps = Arguments.createArray();
-      if (navInfo.getRemainingSteps() != null) {
-        for (StepInfo info : navInfo.getRemainingSteps()) {
-          remainingSteps.pushMap(ObjectTranslationUtil.getMapFromStepInfo(info));
-        }
+    WritableMap map = Arguments.createMap();
+
+    map.putInt("navState", navInfo.getNavState());
+    map.putBoolean("routeChanged", navInfo.getRouteChanged());
+    if (navInfo.getDistanceToCurrentStepMeters() != null)
+      map.putInt("distanceToCurrentStepMeters", navInfo.getDistanceToCurrentStepMeters());
+    if (navInfo.getDistanceToFinalDestinationMeters() != null)
+      map.putInt(
+        "distanceToFinalDestinationMeters", navInfo.getDistanceToFinalDestinationMeters());
+    if (navInfo.getDistanceToNextDestinationMeters() != null)
+      map.putInt("distanceToNextDestinationMeters", navInfo.getDistanceToNextDestinationMeters());
+    if (navInfo.getTimeToCurrentStepSeconds() != null)
+      map.putInt("timeToCurrentStepSeconds", navInfo.getTimeToCurrentStepSeconds());
+    if (navInfo.getTimeToFinalDestinationSeconds() != null)
+      map.putInt("timeToFinalDestinationSeconds", navInfo.getTimeToFinalDestinationSeconds());
+    if (navInfo.getTimeToNextDestinationSeconds() != null)
+      map.putInt("timeToNextDestinationSeconds", navInfo.getTimeToNextDestinationSeconds());
+    if (navInfo.getCurrentStep() != null)
+      map.putMap(
+        "currentStep", ObjectTranslationUtil.getMapFromStepInfo(navInfo.getCurrentStep()));
+
+    WritableArray remainingSteps = Arguments.createArray();
+    if (navInfo.getRemainingSteps() != null) {
+      for (StepInfo info : navInfo.getRemainingSteps()) {
+        remainingSteps.pushMap(ObjectTranslationUtil.getMapFromStepInfo(info));
       }
-      map.putArray("getRemainingSteps", remainingSteps);
+    }
+    map.putArray("getRemainingSteps", remainingSteps);
 
-      WritableNativeArray params = new WritableNativeArray();
-      params.pushMap(map);
-      catalystInstance.callFunction(Constants.NAV_JAVASCRIPT_FLAG, "onTurnByTurn", params);
+    WritableNativeArray params = new WritableNativeArray();
+    params.pushMap(map);
+    catalystInstance.callFunction(Constants.NAV_JAVASCRIPT_FLAG, "onTurnByTurn", params);
```

Comment:
Nit: For future PRs, can we please separate changes that are meant to be mere formatting/styling changes? This will make it easier for reviewers to focus on the functional changes

Reply:
Yes, the project currently lacks formatting configuration, and as I locally have autoformatting, missed these changes. Once the formatting setup is complete and integrated into CI and lefthook linting, these formatting commits should disappear.

In the meantime, you can `Hide whitespace` changes in the GitHub review to focus on meaningful differences.
<img width="253" alt="Screenshot 2024-08-30 at 9 04 34" src="https://github.com/user-attachments/assets/16bee91e-b172-44a4-986e-1f47ed975742">

Reply:
Removed formatting changes on this file for easier review.

Reply:
To avoid these in future, I added forced formatting to the repo for future PR's

https://github.com/googlemaps/react-native-navigation-sdk/pull/240